moon-store-mac-os/moon-store-mac-os/Helpers/Extensions/Data+String.swift

```diff
@@ -10,7 +10,7 @@ import Foundation
 extension Data {
     mutating func appendStringIfNeeded(_ string: String) {
         guard let data = string.data(using: .utf8) else {
-            debugPrint("Could not convert \(string) to UTF8")
+            _ = MSErrorManager(reference: "Data extension").handle(error: .badData)
```

Comment:
What is the reason for this instance?

Reply:
Did you review the comment answer?

Reply:
Doesn't appear

Reply:
I saw that but doesnt make sense for me

Reply:
@Rexmoon You should remove the unnecesary `_ =` because you are saving two call stacks in the deep stack call for swift

Reply:
Method throws an error.

<img width="839" alt="Screenshot 2024-12-13 at 20 42 00" src="https://github.com/user-attachments/assets/d3562fc6-d6a6-4889-8298-c70a2374b98a" />